graphite_beacon/alerts.py

```diff
@@ -128,15 +182,33 @@ def configure(self, name=None, rules=None, query=None, **options):
 
         self.history_size = options.get('history_size', self.reactor.options['history_size'])
         self.history_size = parse_interval(self.history_size)
-        self.history_size = int(math.ceil(self.history_size / interval))
 
         self.no_data = options.get('no_data', self.reactor.options['no_data'])
         self.loading_error = options.get('loading_error', self.reactor.options['loading_error'])
 
-        if self.reactor.options.get('debug'):
-            self.callback = ioloop.PeriodicCallback(self.load, 5000)
+        interval = options.get('interval', self.reactor.options['interval'])
+        time_window = options.get('time_window', None)
+
+        if is_cron(interval):
+            self.interval = interval
```

Comment:
Can you check that this is a valid cron expression before proceeding here? It's best to fail early on bad input